environments/staging/inventory.ini.j2

```diff
@@ -86,9 +83,13 @@ celery1
 [pillow0]
 {{ pillow0-staging }} hostname=pillow0-staging ec2=yes
 
+[pillow1]
+{{ pillow1-staging }} hostname=pillow1-staging ec2=yes
+
 [pillowtop:children]
 # Amazon EC2
 pillow0
+pillow1
```

Comment:
You have to run aws-fill-inventory [--cached] again to sync this change to inventory.ini